src/parser/rogue/subtlety/modules/spell/shadowlands/legendaries/AkaarisSoulFragment.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+import SPELLS from 'common/SPELLS';
+import Events, { DamageEvent } from 'parser/core/Events';
+import { formatNumber } from 'common/format';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+
+class AkaarisSoulFragment extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+  };
+
+  damage: number = 0;
+  protected abilities!: Abilities;
+
+  constructor(options: any) {
+    super(options);
+    // this.active = this.selectedCombatant.hasLegendaryByBonusID();
```

Comment:
should use the commented out line instead of leaving it always active.
this.active = this.selectedCombatant.hasLegendaryByBonusID(SPELLS.AKAARIS_SOUL_FRAGMENT.bonusID);